src/doc/rustc/src/platform-support/avr-unknown-unknown.md

```diff
@@ -0,0 +1,75 @@
+# `avr-unknown-unknown`
+
+Series of microcontrollers from Atmel: ATmega8, ATmega328p etc.
+
+**Tier: 3**
+
+## Target maintainers
+
+-[Patryk Wychowaniec](https://github.com/Patryk27) <pwychowaniec@pm.me>
+
+## Requirements
+
+This target is only cross-compiled - x86_64 / aarch64 x Lixux / MacOS hosts are
+confirmed to work, but in principle any machine able to run rustc and avr-gcc
+should be good.
+
+Compiling for this target requires `avr-gcc`, because a couple of intrinsics
+(like 32-bit multiplication) rely on [`libgcc`](https://github.com/gcc-mirror/gcc/blob/3269a722b7a03613e9c4e2862bc5088c4a17cc11/libgcc/config/avr/lib1funcs.S)
+and can't be provided through `compiler-builtins` yet; this is a limitation that
+we hope to lift in the future.
```

Comment:
Mind opening up an issue at https://github.com/rust-lang/compiler-builtins/issues for what exactly these problems are? Might as well make a todo list for the future.

Reply:
Sure, done; I've also linked the issue in this document.